File: algorithms/2037.minimum-number-of-moves-to-seat-everyone/solution.rs
impl Solution {
    pub fn min_moves_to_seat(seats: Vec<i32>, students: Vec<i32>) -> i32 {
        let (mut seats, mut students) = (seats, students);
        seats.sort_unstable();
        students.sort_unstable();

        let mut res = 0i32;
        for i in 0..seats.len() {
            res += (seats[i] - students[i]).abs();
        }

        res
    }
}

File: algorithms/2037.minimum-number-of-moves-to-seat-everyone/solution.rb
# @param {Integer[]} seats
# @param {Integer[]} students
# @return {Integer}
def min_moves_to_seat(seats, students)
  seats.sort!
  students.sort!

  res = 0
  (0...seats.size).each do |i|
    res += (seats[i] - students[i]).abs
  end

  res
end
